docs/framework/windows-workflow-foundation/deprecatedcontent-md.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Deprecate Content"
+description: DeprecateContent file, content to be included. 
+ms.date: "10/10/2020"
+---
+This material discusses types and namespaces that are obsolete.
```

Comment:
If you want to keep this include file, it needs to go in the /includes directory at the root of this repo. But, I suggest deleting it and just putting the literal text into the article instead. You can still put it in a NOTE block.

Reply:
@gewarren updated.